Extensions/Leaderboards/leaderboardstools.ts

```diff
@@ -52,6 +57,7 @@ namespace gdjs {
               responseVar.setString(data);
             })
             .catch((error) => {
+              console.warn(error);
```

Comment:
Give the context in your messages: `console.warn("Error while submitting a leaderboard score:", error)`